geometric_controller/include/geometric_controller/common.h

```diff
@@ -121,4 +132,171 @@ Eigen::Vector4d rot2Quaternion(const Eigen::Matrix3d &R) {
   return quat;
 }
 
+state_matrix_t A_quadrotor(const state_vector_t& x, const control_vector_t& u);
+control_gain_matrix_t B_quadrotor(const state_vector_t& x, const control_vector_t& u);
+
+state_matrix_t A_quadrotor(const state_vector_t& x, const control_vector_t& u)
```

Comment:
I don't think common is the right place for these matrix definitions. I think this should belong to the controller

Reply:
@brunopinto900 I think we should make one :smile: